src/main/resources/application.properties

```
@@ -17,4 +14,7 @@ jwt.expiration=7200
 
 # Flyway
 spring.flyway.clean-on-validation-error=true
-spring.flyway.locations=classpath:db/migration/COMMON,classpath:db/migration/DEV
\ No newline at end of file
+spring.flyway.locations=classpath:db/migration/COMMON,classpath:db/migration/DEV
```

Nhân tiện

Ở file `application.properties`

```suggestion
spring.flyway.locations=classpath:db/migration/COMMON
```

Ở file

application-DEV.properties

```
spring.flyway.locations=classpath:db/migration/COMMON,classpath:db/migration/DEV
```